typings/interfaces/Permission.d.ts

```diff
@@ -47,6 +47,12 @@ export interface Permission {
    */
   setSkip(value: number): boolean
```

Comment:
`setSkip` is defined twice here, the second one was probably intended to be named `setValue`.